apps/sms/js/thread_ui.js

```diff
@@ -662,50 +660,22 @@ var ThreadUI = {
     }
   },
 
-  handleForward: function thui_handleForward(forward) {
```

Comment:
```handleForward``` uses ```Composer.fromMessage``` as ```handleActivity``` does, I don't see why we can't join them. I also had an idea to rename ```activity``` parameter to smth more generic like ```parameters``` or similar as it can be used not only in activity case (and actually used for ```Send message``` cases). What do you think?